assets/js/modules/ads/datastore/moduleData.test.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * `modules/ads` moduleData store tests.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	createTestRegistry,
+	untilResolved,
+	unsubscribeFromAll,
+} from '../../../../../tests/js/utils';
+import { initialState } from './index';
+import { MODULES_ADS } from './constants';
+
+describe( 'modules/ads module data', () => {
+	const baseModulesDataVar = '_googlesitekitModulesData';
```

Comment:
Let's not abbreviate names, something like:

```suggestion
	const baseModulesDataVariable = '_googlesitekitModulesData';
```

is easier to read.

Better yet is probably just `baseModulesGlobalName`

Reply:
Fair play here, I'll update.